### PR TITLE
Change updates logic

### PR DESCRIPTION
**Pull request by staniel359**

### CODE REVIEW

**Comment by xyloflake** [comment]
@staniel359 I suggest autoupdates on windows to be also enabled as a beta feature. There's no UI so it makes no sense to declare that a stable feature yet.

The ideology is this:
- Win: Stable Beta (Default state on)
- MacOS: Unsupported
- Linux: Experimental/Unstable Beta (Default state off)

File: electron/actions/app/checkForUpdates.js
Comment (xyloflake) [inline electron/actions/app/checkForUpdates.js:130]: Suggest changing that to `willAutoupdate`

File: electron/actions/app/setAutoupdateSetting.js
Comment (xyloflake) [inline electron/actions/app/setAutoupdateSetting.js]: This must be true. Autoupdates don't have the finishing to them yet. 

Main reason being that there's no way for the user to tell if it's downloading the app or not. 

And there's no choice for choosing to autoupdate or declining the update and staying on the initial version.

**Comment by staniel359** [comment]
@xyloflake Updated.

**Review by xyloflake — CHANGES_REQUESTED** [review]
Okay, everything else looks fine. Kindly check the one minor comment. Will approve once that is resolved.

File: electron/actions/app/setAutoupdateSetting.js
Comment (xyloflake) [reply electron/actions/app/setAutoupdateSetting.js]: Resolving because code has been updated. Thanks @staniel359!

**Comment by xyloflake** [comment]
@staniel359 @gamersi should there be a beta test before merging?

File: electron/actions/app/checkForUpdates.js
Comment (staniel359) [reply electron/actions/app/checkForUpdates.js:130]: Sorry, all my boolean variables start with `is`.

**Comment by staniel359** [comment]
@xyloflake

> @staniel359 @gamersi should there be a beta test before merging?

I think so.

**Review by gamersi — APPROVED** [review]
LGTM, let's make a beta test

**Review by xyloflake — APPROVED** [review]
Cool, let's merge only after the beta test. But I'm approving right now. LGTM.

**Pull request merged**
